src/components/LocationAutocomplete/LocationAutocomplete.js

```diff
@@ -0,0 +1,191 @@
+                    types: placeTypes,
+                    componentRestrictions: { country },
+                },
+                (predictions, status) => {
+                    // if this function was called with outdated input value, return early
+                    if (debouncedInputValue !== storage.latestInputValue) return;
+
+                    if (ACCEPTABLE_API_STATUSES.includes(status)) {
+                        setSuggestionsList(predictions);
+                    } else {
+                        // TODO: check desired behaviour with Carlo
+                        // currently the UI will look same as when no suggestions found
+                        resetSuggestionsList();
+                        if (onError) {
+                            onError(status);
+                        }
+                    }
+                    setIsLoading(false);
+                }
+            );
+        } else {
+            resetSuggestionsList();
+        }
+    }, [country, debouncedInputValue, onError, placeTypes, storage.latestInputValue]);
+
+    if (!(window.google && window.google.maps && window.google.maps.places)) {
+        // TODO: clarify with Carlo how to handle errors
+        return (
+            <Alert context="bad" title="No API found">
+                Google Maps Places APi was not found on the page. Before using this component, make
+                sure to load the places API
+            </Alert>
+        );
+    }
+
+    const handleInputValueChange = value => {
+        setInputValue(value);
+        if (value) {
+            setIsLoading(true);
+        } else {
+            setIsLoading(false);
+            resetSuggestionsList();
+        }
+    };
+
+    const handleSelection = value => {
+        resetSuggestionsList();
+        setInputValue('');
+        onSelectionChange(value);
+    };
+
+    // eslint-disable-next-line react/display-name
+    const renderListPoweredByGoogle = ({ listInputValue, getItemProps, highlightedIndex }) => {
+        const elems = suggestionsList.map((item, index) => (
+            <ListItem
+                key={suggestionToString(item)}
+                {...getItemProps({
+                    item,
+                    index,
+                    isHighlighted: highlightedIndex === index,
+                    highlightContext: 'brand',
+                })}
+            >
+                <MarkedText marker={listInputValue} inline>
+                    {showCountryInSuggestions
+                        ? suggestionToString(item)
+                        : item.structured_formatting.main_text}
+                </MarkedText>
+            </ListItem>
+        ));
+
+        if (!hidePoweredByGoogleLogo) {
+            elems.unshift(
+                // TODO: replace this element with correct resolution version
+                // for all available images see: https://developers.google.com/maps/documentation/images/powered_by_google.zip
```

Comment:
will this be addressed? I don't see any blockers for this

Reply:
It isn't 100% straight forward, and since we want to finish this PR and move ahead, and not complicate it even more then what it is now, we said we gonna do it in a separate task.

Reply:
What's the exact difficulty with it? It's just about extracting this piece of code into component and add  a media query for hd display that reference @2x and @3x images from the archive provided.

And if we skip it now, chances high that we'll never have time to come back to it.

Reply:
I can create a ticket now, if it isn't there yet. but there are a lot more then 3 images in that zip folder, and what media query is relevant for each is not that clear

Reply:
I think it's more than enough to add just this one so that it doesn't look blurry on retina.
![image](https://user-images.githubusercontent.com/403250/63700668-af2ec800-c823-11e9-9000-eae4d9d84a09.png)

Reply:
I added retina